gips/data/landsat/landsat.py

```diff
@@ -38,13 +40,16 @@
 # once gippy==1.0, switch to GeoRaster.erode
 from scipy.ndimage import binary_erosion
 
+import osr
 import gippy
 from gips import __version__ as __gips_version__
-from gippy.algorithms import ACCA, Fmask, LinearTransform, Indices, AddShadowMask
+from gips.core import SpatialExtent, TemporalExtent
+from gippy.algorithms import ACCA, Fmask, LinearTransform, Indices, AddShadowMask, CookieCutter
 from gips.data.core import Repository, Asset, Data
 from gips.atmosphere import SIXS, MODTRAN
 import gips.atmosphere
-from gips.utils import RemoveFiles, basename, settings, verbose_out
+from gips.inventory import DataInventory
+from gips.utils import RemoveFiles, basename, settings, verbose_out, import_data_class, make_temp_dir, open_vector, transform_shape
```

Comment:
We do have a dev practices document in `Drive >> Projects >> GIPS`, which is essentially GitFlow+PEP8.  PEP8 defers to [80 columns still](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).  If we want to have a discussion around modifying that, let's have an off-line discussion.

Reply:
I don't suggest a change in standared; I merely suggest that it's good to call out line length explicitly, because 80 columns is a lot less common in the industry these days (and clearly @naterubin didn't know).  I myself only realized we prefer <80 columns after many months, when I noticed you reformatting some code I'd written.  : ]